po/es.po

```diff
@@ -0,0 +1,747 @@
+# Spanish translations for connman-gtk package.
+# Copyright (C) 2015 THE connman-gtk'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the connman-gtk package.
+# Isaac Quiroz <isaac.qa13@gmail.com>, 2018.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: connman-gtk 1.1\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-06-11 07:20+0400\n"
+"PO-Revision-Date: 2018-06-11 09:59-0400\n"
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: Maloqui <maloqui@Void4233>\n"
+"Language-Team: Spanish\n"
+"X-Generator: Poedit 2.0.7\n"
+
+#: src/agent.c:158
+msgid "Authentication required"
+msgstr "Autenticación requerida"
+
+#: src/config.c:62
+msgid "Use fsid with openconnect"
+msgstr "Usar fsis con openconnect"
```

Comment:
Should this be also 'fsid'? I don't speak Spanish, but I think it should be written the same regardless of the language (since it's 'file system id' or something like that')